Add connect to wallet in swap (#217)
cowprotocol/docs: docs/cow-protocol/tutorials/cow-swap/swap.mdx

Swapping is the bread and butter of CoW Protocol! Let's swap some tokens!

> ## Connect Wallet

Reviewer: ```suggestion
## Connect wallet
```

> Go to [swap.cow.fi](https://swap.cow.fi/#/swap?utm_source=docs.cow.fi&utm_content=swap-tutorial) and connect your web3 wallet. 

Reviewer: ```suggestion
Go to [CoW Swap](https://swap.cow.fi/#/swap?utm_source=docs.cow.fi&utm_content=swap-tutorial) and connect your web3 wallet. 
```

:::note

> If you don't have a web3 wallet yet, you can easily get one!

Reviewer 2: ```suggestion
If you don't have a web3 wallet yet, you can easily get one! Here are some suggestions:
```

> - Use a browser extension wallet such as [Rabby](https://rabby.io/) or [MetaMask](https://metamask.io/download.html).

Reviewer: ```suggestion
- Use a browser extension wallet such as [Rabby](https://rabby.io/) or [Metamask](https://metamask.io/download.html)
```

Suggest some standardisation on the capitalisation of Metamask (not concerned which way)

> - A hardware wallet such as [Ledger](https://www.ledger.com/) or [Trezor](https://trezor.io/).

Reviewer: ```suggestion
- A hardware wallet such as [Ledger](https://www.ledger.com/) or [Trezor](https://trezor.io/)
```

Author: I will ask Alex M what he thinks about this :)

- A Smart Contract Wallet such as [Safe](https://app.safe.global)

> Please do your own research on the trade offs of each wallet to select the one that works best for you.

Reviewer: ```suggestion
Please do your own research on the advantages and disadvantages of each wallet to select the one that works best for you.
```

nit: above suggestion to get away from "trade off" which can be contentious about whether it is hyphenated or not :laughing:
